solr/core/src/java/org/apache/solr/util/DOMUtil.java

```diff
@@ -39,9 +42,23 @@
 
   public static final String XML_RESERVED_PREFIX = "xml";
 
+  static final Set<String>  NL_TAGS = ImmutableSet.of("str", "int","long","float","double","bool");
```

Comment:
can we use Set.of instead of ImmutableSet

Reply:
`Set.of()` is java 9. I'm hoping to put this into Solr 8.x